### DISCUSSION TITLE
Cocotb error with Verilator

### DISCUSSION THREAD

**Discussion by CCCDDDEEEFFF** [cocotb/cocotb#3602] [Q&A] [answered]
I am trying to run the example in the Quick start with Verilator but am running into this issue. 
Cocotb version 1.5.2
Verilator version 4.106
[Quickstart](https://docs.cocotb.org/en/stable/quickstart.html)

```
/usr/local/lib/python3.8/dist-packages/cocotb/share/lib/verilator/verilator.cpp:117:18: error: 'class Vtop' has no member named 'eventsPending'
  117 |             top->eventsPending() ? top->nextTimeSlot() : NO_TOP_EVENTS_PENDING;
      |                  ^~~~~~~~~~~~~
/usr/local/lib/python3.8/dist-packages/cocotb/share/lib/verilator/verilator.cpp:117:41: error: 'class Vtop' has no member named 'nextTimeSlot'
  117 |             top->eventsPending() ? top->nextTimeSlot() : NO_TOP_EVENTS_PENDING;
```

**Comment by ktbarrett** [comment, marked as answer]
Use the newest version of Verilator available. The issue causing cocotb to not work with anything besides v4.106 has been resolved and the minimum version supported is now [5.006](https://docs.cocotb.org/en/stable/simulator_support.html#verilator).
